packages/@okta/vuepress-site/docs/guides/sign-into-spa/add-signin-button/react/login-redirect.md

```diff
@@ -1,48 +1,68 @@
-The `Auth` object is provided on your component's props via the `withAuth` component. You can use the `auth.isAuthenticated()` method to show or hide a button depending on whether the user is signed in.
+The `authState` object provides information on the state of the current user's authentiation.  
```

Comment:
If possible, I think it would be good to preface this list of notes on specific objects with at least one intro sentence that just generally says that this section of the guide is about tools available for you to use in React when implementing a sign in button.